### DISCUSSION TITLE
Blink 14 Upgrade

### DISCUSSION THREAD

**Discussion by rwarner** [blinksh/blink#1760] [Q&A] [answered]
Sent this via email a little bit ago, but thought I'd post here as well.

I've been using Blink 14 for some time now. However, I noticed recently there's a bit of weirdness happening with the appearance going on with the cursor and text being cut off. Upon checking the settings I noticed the Blink Upgrade expiring soon. I had seen the notification a long time ago but didn't completely understand the difference between 14 and the new product.

If I wanted to take advantage of the upgrade is that still possible? When I clicked the button around the date of the 15th it brought me to the new app in the App Store and nothing happenned. When I download the app and go through the process I don't see any sort of redemption. I understand it's past the deadline but thought I'd ask.

**Comment by carloscabanero** [comment, marked as answer]
Hey, yes we are still trying to help those we can. I thought I was all caught up on that but sorry if I missed your email. Would you mind to send me a ping on that email thread?

**Reply by rwarner** [reply]
Hey Carlos, thanks for the quick answer! Will re-send the email

**Reply by Qtechnetita** [reply]
Hi Carlos I’m an earlier and happier 14 version user but I miss the upgrade to 15 for free time window. Is there any chance to get the new and supported version at this time? Many thanks

**Reply by carloscabanero** [reply]
Yes, we are trying to help as many people as we can. Jusy reach out to support at blink.sh.

**Reply by Qtechnetita** [reply]
Many thanks Carlos! Sorry for stupid question would you send me the exact link of support page at blink.sh (I wasn't able to find it...)

**Reply by rwarner** [reply]
@Qtechnetita email them `support@blink.sh`
